detox/test/e2e/17.datePicker.test.js

```diff
@@ -3,12 +3,17 @@ describe(':ios: DatePicker', () => {
       await device.reloadReactNative();
       await element(by.text('DatePicker')).tap();
     });
-    
+
     it('datePicker should trigger change handler correctly', async () => {
       await element(by.type('UIPickerView')).setColumnToValue(1,"6");
       await element(by.type('UIPickerView')).setColumnToValue(2,"34");
       await expect(element(by.id("timeLabel"))).toHaveText('choosenTime is 6:34');
     });
 
-  });
-  
+    it('can select dates on a UIDatePicker', async () => {
+      await element(by.type('UIDatePicker')).setDatePickerDateIOSOnly('2019-02-06T05:10:00-08:00', "yyyy-MM-dd'T'HH:mm:ssZZZZZ");
```

Comment:
This should be `setDatePickerDate` and documentation should be updated to state it is for iOS only, as are the current picker view APIs.

Reply:
📗 9470d65